### DISCUSSION TITLE
Small improvments for gridmaps

### DISCUSSION THREAD

**Discussion by cyprien-griot** [IPGP/webobs#127] [Ideas]
Greetings all,

After some discussions with Aline who uses gridmaps outputs for communication purposes (reports, etc.), we thought about some possible improvments :

- Making gridmaps job produces 2 maps : one with both active and inactive nodes, one with only active ones. I know we could use 2 different views, but this would implies double work with nodes management (I am not sure if what I mean is clear but I could explain more if needed). A map with only active nodes would look "prettier" and less confusing for official communication. 

- Would it be possible to make the legend consistent, depending on the zoom ? What I mean is, at the moment when we have X nodes displayed on map0, legend shows X nodes. But on map1, which shows Y nodes, legend still shows X nodes, instead of Y. If legend could show Y, it would be more consistent.

What do you think ? 

Regards,

**Comment by beaudu** [comment]
* first point: there is an undocumented option in GRIDMAPS.rc to show/hide inactive nodes in grid maps. When the gridmaps request tool will be fully functional, a manual request will produce any map for a selected period of time with/without inactive node at that time.
* second point: actually it was a real bug. The commit b494ff6 should fix it.
